src/pytom_tm/tmjob.py

```diff
@@ -233,18 +235,33 @@ def __init__(
         self.start_slice = 0
         self.steps_slice = 1
         self.rotational_symmetry = rotational_symmetry
+        self.angle_increment = angle_increment
+        succesfull_angle = False
         try:
-            self.rotation_file = AVAILABLE_ROTATIONAL_SAMPLING[angle_increment][0]
-            self.n_rotations = AVAILABLE_ROTATIONAL_SAMPLING[angle_increment][1]
-        except KeyError:
+            angle_increment = float(angle_increment)
+            angle_is_float = True
+        except (ValueError, TypeError):
+            angle_is_float = False
+        if angle_is_float:
+            # Generate angle list on the fly
+            logging.info(f"Will generate an angle list with a maximum increment of {angle_increment}")
+            self.rotation_file = None
+            self.angle_list = angle_to_angle_list(angle_increment)
```

Comment:
The list is now regenerated when loading a job file for extraction, fine by me as its fast (although the printing to terminal might be confusing). Potentially, could make use of the keyword `job_loaded_for_extraction` that I introduced for a similar issue to skip it?

Reply:
The full angle_list is now also written to the json file. That seems a bit unnecessary and makes the json job file difficult to interpret. As the generation is anyway fast and reproducible, maybe we can skip saving it ?

Might be easier to generate the list in TMJob.start_job() ? That saves the extraction recalculation and prevents saving to file

Reply:
I decided to regenerate it on the fly instead of storing it to json and also restructured the way the loading logging is done